utils/rpms/daos.spec

```
@@ -213,13 +220,15 @@ scons %{?_smp_mflags}                 \
       --install-sandbox=%{?buildroot} \
       %{?buildroot}%{_prefix}         \
       %{?buildroot}%{conf_dir}        \
-      USE_INSTALLED=all               \
       CONF_DIR=%{conf_dir}            \
       PREFIX=%{_prefix}               \
-      %{?scons_args}
+      %{?scons_args}                  \
```

I wonder if the `%install` `scons` is rebuilding because it does not have the same `COMPILER=` argument as the `%build` `scons`.  I think you might want to add the `%{?compiler_args}` args here:
```suggestion
      %{?scons_args}                  \
      %{?compiler_args}
```